oamapps/postConfigure/helpers.cpp

```diff
@@ -728,12 +740,14 @@ void checkMysqlPort( std::string& mysqlPort, Config* sysConfig )
 {
     Oam oam;
 
+	string tmpDir = startup::StartUp::tmpDir();
+
     while (true)
```

Comment:
this looks like an infinite loop that always exits after the first iteration...?